### PR TITLE
Added tests for functions in src/python.py

### PR DESCRIPTION
**Pull request by sudiptog81**
Added unit tests for `get_int()`, `get_float()` and `get_string()` in `test/python.py`. These tests produce an `AssertionError` when these functions return values whose types differ from expected types.

### CODE REVIEW

File: tests/python.py [hunk 1]
```diff
@@ -4,5 +4,29 @@
...
+def test_get_int():
+    """
+    Tests the get_int() function
+    """
+    i = cs50.get_int("Integer: ")
+    assert type(i) == type(0), "get_int() did not return an integer"
+
+def test_get_float():
+    """
+    Tests the get_ifloat() function
```
Comment (sudiptog81) [inline tests/python.py]: fixed the typo in the next commit

**Comment by dmalan** [comment]
Happy to integrate tests into the library, but these don't seem to be (automated) unit tests? Each test seems to require that a human input a value when prompted by `get_*`?

**Comment by sudiptog81** [comment]
Yes, it requires user inputs for each of the three function calls. Have to mock the calls using the `unittest` module. Working on that!

**Comment by sudiptog81** [comment]
Have automated the tests and the latest Travis build containing those tests is successful.

**Comment by kzidane** [comment]
I'm afraid that seems to be testing a mocked versions of the `get_*` functions, and not actually testing the functions themselves!

**Pull request closed without merging**